packages/@markuplint/types/src/whatwg/check-autocomplete.spec.ts

```diff
@@ -369,9 +369,9 @@ test('duplicated part', () => {
 });
 
 test('typo', () => {
-	expect(check('secsion-foo name')).toStrictEqual({
+	expect(check('section-foo name')).toStrictEqual({
 		matched: false,
-		raw: 'secsion-foo',
+		raw: 'section-foo',
```

Comment:
@undsoft 

This is purposely for the misspelling test.
Can you revert it?

Reply:
Yeah, sorry, forgot to fix this test after running.

Reply:
Thank you!